CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#1407](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1407))
 - `opentelemetry-instrumentation-logging` Add `otelTraceSampled` to instrumetation-logging
   ([#1773](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1773))
+- Make Flask request span attributes available for `start_span`. 
```

Comment:
Please move this entry to the Unreleased section

Reply:
Moved in [96bf758](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1784/commits/96bf758a8005d90b06b3b63980f4b7b0a2d461dc).